openstack/networking/v2/extensions/portsecurity/requests.go

```diff
@@ -0,0 +1,55 @@
+package portsecurity
+
+import (
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/ports"
+)
+
+// PortCreateOptsExt adds port security options to the base ports.CreateOpts.
+type PortCreateOptsExt struct {
+	ports.CreateOptsBuilder
+
+	// PortSecurityEnabled toggles port security on a port.
+	PortSecurityEnabled *bool `json:"port_security_enabled,omitempty"`
```

Comment:
The reason I chose `*bool` here is because when the port security extension is enabled, port security is turned on by default. `*bool` will prevent accidentally turning off port security.